### PR TITLE
Add heading for introduction, change numbering, add borders to images

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

**Comment by Luyao-Zhang-1** [comment]
@Adam-Dupaski could you help me review this PR?

File: content/appstore/modules/web-actions.md [hunk 1]
```diff
@@ -16,9 +18,9 @@ The [Web Actions](https://marketplace.mendix.com/link/component/114337/) module
 | Set focus | Sets the focus to the element found with the selector. The element should be able to hold focus like a link, button, or input. For this action, you need to provide a target selector. This target could be an element containing a class (for example, `.mx-class-name`, `.my-class-name`), an HTML element (for example, `td`, `div`, `a`), or an element ID (for example, `#my-element-id`). |
 | Take picture | Allows users to take a picture from the back and front camera in a web or PWA app. |
 
-## 1 Usage
+## 2 Usage
 
-### 1.1 Take Picture
+### 2.1 Take Picture
```
Comment (Adam-Dupaski) [inline content/appstore/modules/web-actions.md:23]: If it's just one sub-section, it's generally not necessary. You can change heading 2.1 into an intro sentence: "To take a picture..."

Comment (Luyao-Zhang-1) [reply content/appstore/modules/web-actions.md]: Indeed. Done.

**Review by Adam-Dupaski — APPROVED** [review]

**Comment by Adam-Dupaski** [comment]
Looks good to merge.

**Pull request merged**